winlogbeat/etc/fields.yml

```diff
@@ -164,6 +221,11 @@ eventlog:
       description: >
         The type of account associated with this event.
 
+    - name: version
+      type: int
```

Comment:
I will fix these is a separate PR. This will change the template and mapping for these fields (long -> integer) so I want to make sure there are no unintended consequences.